schemahcl/stdlib_test.go

```diff
@@ -197,6 +197,26 @@ func TestMakeGlobFunc(t *testing.T) {
 	require.Equal(t, []string{"a.hcl", "b.hcl", "variables.hcl"}, result)
 }
 
+func TestMakeFilesetFunc(t *testing.T) {
+	fn := MakeFileSetFunc("testdata")
+	_, err := fn.Call([]cty.Value{cty.StringVal("foo")})
+	require.EqualError(t, err, "base directory must be an absolute path. got: testdata")
+
+	base, err := filepath.Abs("testdata")
+	require.NoError(t, err)
+	fn = MakeFileSetFunc(base)
+	v, err := fn.Call([]cty.Value{cty.StringVal("*.hcl")})
```

Comment:
Can you please add test for nested case?

Reply:
Sure